config/default.json

```diff
@@ -38,7 +38,13 @@
   "RUN_MIGRATIONS": true,
   "ENDPOINT_CACHE_CONFIG": {
     "expiresIn": 180000,
-    "generateTimeout": 30000
+    "generateTimeout": 30000,
+    "getDecoratedValue": true
```

Comment:
Hi @kleyow, can you explain about this new configuration parameter? I observed this in shared lib also. May be you can add some documentation about this in README.

Reply:
done. @vijayg10